content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -222,15 +175,14 @@ System.in specifies that we are getting the keyboard input.
   
 ### Full Client-Side Application
```

Comment:
Change:

> \#\#\# Full Client-Side Application

To:

> \#\#\#\# Full Client-Side Application

I mentioned this last time but it seems you forgot to fix this.